openupgradelib/openupgrade.py

```diff
@@ -26,11 +26,16 @@
 from openerp import release, tools, SUPERUSER_ID
 from openerp.tools.yaml_import import yaml_import
 from openerp.osv import orm
-from openerp.tools.mail import plaintext2html
 from openerp.modules.registry import RegistryManager
 import openerp.osv.fields
-import openerp.fields
 from . import openupgrade_tools
+if release >= "7.0":
+    from openerp.tools.mail import plaintext2html
+if release >= "8.0":
+    from openerp.fields.Many2many import Many2many, One2many
```

Comment:
You're not importing other variables... `release` is a module which contains `version_info`. `release` is already imported

It would be incredibly irresponsible to change these internals since their purpose is to not change.

I'm not even convinced that this change works.

To make sure, could you change these separate if statements to be `if.. elif..else` where the else warns that no version was found.